src/Common/tests/System/Runtime/Serialization/Formatters/BinaryFormatterHelpers.cs

```diff
@@ -30,13 +26,6 @@ internal static T Clone<T>(T obj)
 
         internal static Lazy<T> Clone<T>(Lazy<T> lazy)
```

Comment:
I think you can remove this overload entirely now. It was only needed for the .NETNative escape path.